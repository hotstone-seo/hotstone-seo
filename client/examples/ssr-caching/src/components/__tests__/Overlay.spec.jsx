import React from 'react';
import Overlay from '../Overlay.jsx';

test('Should render Overlay correctly', () => {
  const tree = render(
    <Overlay>
      <div className="test">Overlay</div>
    </Overlay>
  );

  expect(tree).toMatchSnapshot();
});
