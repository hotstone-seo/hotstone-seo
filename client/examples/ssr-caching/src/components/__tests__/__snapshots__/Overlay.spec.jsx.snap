// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Overlay correctly 1`] = `null`;
